discord/permissions.go

```diff
@@ -117,45 +121,47 @@ const (
-	PermissionVoiceDeafenMembers:      "Deafen Members",
-	PermissionVoiceMoveMembers:        "Move Members",
-	PermissionVoiceUseVAD:             "Use Voice Activity",
-	PermissionVoicePrioritySpeaker:    "Priority Speaker",
-	PermissionChangeNickname:          "Change Nickname",
-	PermissionManageNicknames:         "Manage Nicknames",
-	PermissionManageRoles:             "Manage Roles",
-	PermissionManageWebhooks:          "Manage Webhooks",
-	PermissionManageEmojisAndStickers: "Manage Emojis and Stickers",
-	PermissionUseApplicationCommands:  "Use Application Commands",
-	PermissionRequestToSpeak:          "Request to Speak",
-	PermissionManageEvents:            "Manage Events",
-	PermissionManageThreads:           "Manage Threads",
-	PermissionCreatePublicThread:      "Create Public Threads",
-	PermissionCreatePrivateThread:     "Create Private Threads",
-	PermissionUseExternalStickers:     "Use External Stickers",
-	PermissionSendMessagesInThreads:   "Send Messages in Threads",
-	PermissionStartEmbeddedActivities: "Start Embedded Activities",
-	PermissionModerateMembers:         "Moderate Members",
+	PermissionCreateInstantInvite:              "Create Instant Invite",
+	PermissionKickMembers:                      "Kick Members",
+	PermissionBanMembers:                       "Ban Members",
+	PermissionAdministrator:                    "Administrator",
+	PermissionManageChannels:                   "Manage Channels",
+	PermissionManageServer:                     "Manage Server",
+	PermissionAddReactions:                     "Add Reactions",
+	PermissionViewAuditLogs:                    "View Audit Logs",
+	PermissionViewChannel:                      "View Channel",
+	PermissionSendMessages:                     "Send Messages",
+	PermissionSendTTSMessages:                  "Send TTS Messages",
+	PermissionManageMessages:                   "Manage Messages",
+	PermissionEmbedLinks:                       "Embed Links",
+	PermissionAttachFiles:                      "Attach Files",
+	PermissionReadMessageHistory:               "Read Message History",
+	PermissionMentionEveryone:                  "Mention Everyone",
+	PermissionUseExternalEmojis:                "Use External Emojis",
+	PermissionVoiceConnect:                     "Connect",
+	PermissionVoiceSpeak:                       "Speak",
+	PermissionVoiceMuteMembers:                 "Mute Members",
+	PermissionVoiceDeafenMembers:               "Deafen Members",
+	PermissionVoiceMoveMembers:                 "Move Members",
+	PermissionVoiceUseVAD:                      "Use Voice Activity",
+	PermissionVoicePrioritySpeaker:             "Priority Speaker",
+	PermissionChangeNickname:                   "Change Nickname",
+	PermissionManageNicknames:                  "Manage Nicknames",
+	PermissionManageRoles:                      "Manage Roles",
+	PermissionManageWebhooks:                   "Manage Webhooks",
+	PermissionManageGuildExpressions:           "Manage Emojis and Stickers",
```

Comment:
this will probably need to be changed as well but it's currently like this in the client